netbout-web/src/main/java/com/netbout/dynamo/DyAttachments.java

```diff
@@ -236,6 +241,7 @@ public void create(final String name) throws IOException {
                 .with(DyAttachments.ATTR_CTYPE, "text/plain")
                 .with(DyAttachments.ATTR_DATA, " ")
                 .with(DyAttachments.ATTR_ETAG, "empty")
+                .with(DyAttachments.ATTR_DATE, System.currentTimeMillis())
```

Comment:
@CarlosGines please inline `"date"` here, remove the public static literal and handle the test accordingly. See [here](http://www.yegor256.com/2015/07/06/public-static-literals.html) for suggestions on how to do it right.